### ISSUE TITLE
Error!!  (Pyfa told me to report the error and post the log here)

### ISSUE THREAD

**Issue by Davak-Kateelo** [pyfa-org/Pyfa#2261]
## Bug Report
```
pyfa v2.27.0
EVE Data Version: 1792574 (2020-08-25 04:10:32)

OS version: Windows-10-10.0.18362-SP0
Python version: 3.7.8 (tags/v3.7.8:4b47a5b6ba, Jun 28 2020, 08:53:46) [MSC v.1916 64 bit (AMD64)]
wxPython version: 4.0.6 (wxWidgets 3.0.5)
SQLAlchemy version: 1.3.19
Logbook version: 1.5.3
Requests version: 2.24.0
Dateutil version: 2.8.1

####################

Traceback (most recent call last):
  File "gui\builtinShipBrowser\sfBrowserItem.py", line 285, in OnPaint
wx._core.wxAssertionError: C++ assertion "!bitmap.GetSelectedInto() || (bitmap.GetSelectedInto() == GetOwner())" failed at ..\..\src\msw\dcmemory.cpp(116) in wxMemoryDCImpl::DoSelect(): Bitmap is selected in another wxMemoryDC, delete the first wxMemoryDC or use SelectObject(NULL)

####################

Traceback (most recent call last):
  File "gui\builtinShipBrowser\sfBrowserItem.py", line 292, in OnPaint
  File "gui\builtinShipBrowser\sfBrowserItem.py", line 236, in Render
wx._core.wxAssertionError: C++ assertion "bmp.IsOk()" failed at ..\..\src\msw\dc.cpp(1313) in wxMSWDCImpl::DoDrawBitmap(): invalid bitmap in wxMSWDCImpl::DrawBitmap

####################

Traceback (most recent call last):
  File "gui\builtinShipBrowser\sfBrowserItem.py", line 292, in OnPaint
  File "gui\builtinShipBrowser\sfBrowserItem.py", line 236, in Render
wx._core.wxAssertionError: C++ assertion "bmp.IsOk()" failed at ..\..\src\msw\dc.cpp(1313) in wxMSWDCImpl::DoDrawBitmap(): invalid bitmap in wxMSWDCImpl::DrawBitmap

####################

Traceback (most recent call last):
  File "gui\builtinShipBrowser\sfBrowserItem.py", line 292, in OnPaint
  File "gui\builtinShipBrowser\sfBrowserItem.py", line 236, in Render
wx._core.wxAssertionError: C++ assertion "bmp.IsOk()" failed at ..\..\src\msw\dc.cpp(1313) in wxMSWDCImpl::DoDrawBitmap(): invalid bitmap in wxMSWDCImpl::DrawBitmap

```

### Expected behavior:
Searching "!!!" should pull up all fits that start with !!!

### Actual behavior:
A couple fits starting with "!!!" may show but then Pyfa slows, stutters or crashes.

### Detailed steps to reproduce:
Name a fit to start with  "!!!" .   Close, reopen Pyfa.  Click on "Fittings" tab; click in the search bar; type "!!!".  About 40% of the time, error occurs that causes Pyfa to stutter or crash.

### Release or development git branch? Please note the release version or commit hash:
Python version: 3.7.8

### Operating system and version (eg: Windows 10, OS X 10.9, OS X 10.11, Ubuntu 16.10):

OS version: Windows-10-10.0.18362-SP0

### Other relevant information:
First time posting an error report here; not very IT inclined so be gentle if my formatting/report isn't the best ;)

Thank you for your help; and for producing such a great tool!

**Comment by blitzmann** [comment]
I personally cannot reproduce the issue. How many fits do you have that start with `!!!`? Can you send me your `saveddata.db` file? You can find it by going to `%userprofile%/.pyfa` in File Explorer. Send it to `holmes.ryan.90+pyfa@gmail.com`

This seems like it's related to #2121 but the error is different and you're not getting a bajillion of them.

**Comment by JohnEuler** [comment]
I get a similar error. Could it be related?

pyfa v2.30.0
EVE Data Version: 1832008 (2020-10-27 09:28:42)

OS version: Windows-10-10.0.19041-SP0
Python version: 3.7.9 (tags/v3.7.9:13c94747c7, Aug 17 2020, 18:58:18) [MSC v.1900 64 bit (AMD64)]
wxPython version: 4.0.6 (wxWidgets 3.0.5)
SQLAlchemy version: 1.3.20
Logbook version: 1.5.3
Requests version: 2.24.0
Dateutil version: 2.8.1

####################

Traceback (most recent call last):
  File "gui\builtinShipBrowser\sfBrowserItem.py", line 285, in OnPaint
wx._core.wxAssertionError: C++ assertion "!bitmap.GetSelectedInto() || (bitmap.GetSelectedInto() == GetOwner())" failed at ..\..\src\msw\dcmemory.cpp(116) in wxMemoryDCImpl::DoSelect(): Bitmap is selected in another wxMemoryDC, delete the first wxMemoryDC or use SelectObject(NULL)

**Comment by DarkFenX** [comment]
Allegedly fixed by #2629
